charts/k8s-monitoring/charts/feature-annotation-autodiscovery/README.md

```diff
@@ -57,6 +57,7 @@ Be sure perform actual integration testing in a live environment in the main [k8
 |-----|------|---------|-------------|
 | annotations.instance | string | `"k8s.grafana.com/instance"` | Annotation for overriding the instance label |
 | annotations.job | string | `"k8s.grafana.com/job"` | Annotation for overriding the job label |
+| annotations.metricsParam | string | `"k8s.grafana.com/metrics.param"` | Annotation for setting __param_<key> parameters when scraping. Example: `k8s.grafana.com/metrics.param_key: "value"`. |
```

Comment:
I don't understand what this is used for, what sort of key/values would be used here?

Reply:
It's a bit of a niche feature, but it allows you to provide `__param_<name>` labels (https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config), which lets you add URL parameters to the request.

i.e. This annotation:
```
metadata:
  annotation:
    k8s.grafana.com/metrics.param_color: blue
```

Will mean that the prometheus.scrape will use the URL `http://<target host>/metrics?color=blue`

Reply:
I've seen it made available in other annotation autodiscovery features and I wanted to have parity.